### PR TITLE
Fixed MPOW-480: Authorize net 2.4.0 card is not saved on subscription creation

### PR DESCRIPTION
**Pull request by alexkwellbe**

### CODE REVIEW

File: Model/Source/Cctype.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
```
Comment (vsevolod-tarasov) [inline Model/Source/Cctype.php:2]: licenses missing

File: Model/Source/Cctype.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+namespace TNW\AuthorizeCim\Model\Source;
+
+use Magento\Payment\Model\Source\Cctype as PaymentCctype;
+
+class Cctype extends PaymentCctype
```
Comment (vsevolod-tarasov) [inline Model/Source/Cctype.php:7]: phpdocs missing

File: Model/Source/Cctype.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+{
+    /**
+     * @inheritdoc
+     */
+    public function getAllowedTypes()
+    {
+        return ['VI', 'MC', 'AE', 'DI', 'JCB', 'DN'];
```
Comment (vsevolod-tarasov) [inline Model/Source/Cctype.php:14]: shouldn`t those been taken from config?

File: Model/Source/Cctype.php (hunk 2 above)
Comment (alexkwellbe) [reply Model/Source/Cctype.php:13]: done

File: Model/Source/Cctype.php (hunk 1 above)
Comment (alexkwellbe) [reply Model/Source/Cctype.php]: done

File: Model/Source/Cctype.php (hunk 3 above)
Comment (alexkwellbe) [reply Model/Source/Cctype.php:20]: It is only a source model for allowed card types in admin config.

**Pull request merged**